.github/workflows/check_with_databases.yml

```diff
@@ -105,7 +106,12 @@ jobs:
         uses: r-lib/actions/setup-r-dependencies@v2
         with:
           # Necessary to avoid object usage linter errors.
-          extra-packages: local::.
+          extra-packages: |
+            local::.
+            any::rcmdcheck
+            any::spelling
```

Comment:
Are those needed in `pre-commit` setup? If yes, please add them to the contribution guide.

Reply:
I believe they are not needed there.

All but `rcmdcheck` and `covr` should be used in `precommit` and it doesn't have any hooks for these 2. (spelling has its own hook that installs it internally)